proto/src/confstate.rs

```diff
@@ -17,7 +17,18 @@ fn eq_without_order(lhs: &[u64], rhs: &[u64]) -> bool {
 }
 
 // Returns true if the inputs describe the same configuration.
+#[must_use]
 pub fn conf_state_eq(lhs: &ConfState, rhs: &ConfState) -> bool {
+    // In most cases, the orders are the same.
```

Comment:
Maybe needs a more precise description like in what cases the orders aren't same